### PR TITLE
Lint all example projects

### PR DESCRIPTION
**Pull request by dfeinzimer**
- Adds the same "Lint Sources" build phase used by the main Examples project to the other two more recent example projects
- Resolves newly raised warnings and some other simple issues

### CODE REVIEW

**Comment by dfeinzimer** [comment]
Waiting to merge #688 and #689 first to avoid conflicts and pickup the new sorted imports rule

File: AuthenticationExample/AuthenticationExample/ProfileView.swift [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
-import SwiftUI
 import ArcGIS
 import ArcGISToolkit
+import SwiftUI
 
 /// A view that displays the profile of a user.
 struct ProfileView: View {
     /// The portal that the user is signed in to.
-    @State var portal: Portal
+    let portal: Portal
```
Comment (dfeinzimer) [inline AuthenticationExample/AuthenticationExample/ProfileView.swift:22]: This is set externally and never modified internally.

File: AuthenticationExample/AuthenticationExample/UserView.swift [hunk 2]
```diff
@@ -12,13 +12,13 @@
...
 
-import SwiftUI
 import ArcGIS
+import SwiftUI
 
 /// A view that displays details of a portal user.
 struct UserView: View {
     /// The user to display the details for.
-    @State var user: PortalUser
+    let user: PortalUser
```
Comment (dfeinzimer) [inline AuthenticationExample/AuthenticationExample/UserView.swift:21]: This is set externally and never modified internally.

**Comment by dfeinzimer** [comment]
@njarecha This is now ready for review, thanks!

**Review by njarecha — APPROVED** [review]
Looks good

**Review by mhdostal — APPROVED** [review]

**Pull request merged**